src/core/style/expressions/quantiles.js

```diff
@@ -54,6 +54,8 @@ export default class Quantiles extends Expression {
         const total = column.accumHistogram[999];
         const r = Math.random();
         let brs = [];
+
+        // TODO OPT: this could be faster with binary search
         this.breakpoints.map((breakpoint, index) => {
             for (i; i < 1000; i++) {
```

Comment:
why 1000? 

https://refactoring.guru/replace-magic-number-with-symbolic-constant

Reply:
Yep, no need for the link :slightly_smiling_face: